test_util/marathon.py

```diff
@@ -202,6 +202,85 @@ def _pool_for_marathon_app(app_id):
             raise Exception("Application deployment failed - operation was not "
                             "completed in {} seconds.".format(timeout))
 
+    def deploy_test_pod_and_check(self, pod, test_uuid):
+        with deploy_pod_and_cleanup(pod) as instances:
+            assert pod['scaling']['instances'] == instances
+
+    def deploy_pod(self, pod_definition, timeout=300, check_health=False):
```

Comment:
Do you know if pods take significantly longer to deploy? we set timeouts to 120s for apps and would be nice to minimize this from the start